<launch>

  <group ns="arm1">
    <node pkg="eezy_arm_ros" name="controller" type="controller.py"/>

    <rosparam param="ports">[2, 4, 17, 22]</rosparam>
    <rosparam param="jt_offsets">[0, 5, -8]</rosparam>
    <rosparam param="ee_open">0.0</rosparam>
    <rosparam param="ee_closed">-90.0</rosparam>
  </group>

</launch>
